src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ClientSkinSystem.java

```diff
@@ -15,38 +15,37 @@
  */
 package org.terasology.ligthandshadow.componentsystem.controllers;
 
-import org.terasology.entitySystem.entity.EntityBuilder;
-import org.terasology.entitySystem.entity.EntityManager;
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.ReceiveEvent;
 import org.terasology.entitySystem.systems.BaseComponentSystem;
 import org.terasology.entitySystem.systems.RegisterMode;
 import org.terasology.entitySystem.systems.RegisterSystem;
 import org.terasology.ligthandshadow.componentsystem.LASUtils;
 import org.terasology.ligthandshadow.componentsystem.events.AddPlayerSkinToPlayerEvent;
-import org.terasology.logic.location.LocationComponent;
-import org.terasology.registry.In;
+import org.terasology.logic.characters.VisualCharacterComponent;
+import org.terasology.rendering.logic.SkeletalMeshComponent;
+import org.terasology.utilities.Assets;
 
```

Comment:
Could you add the class Javadoc too?

Reply:
Added in other PR. It would lead to a merge conflict if I added it to both.